frontend/src/components/Selectors/StateSelector.tsx

```diff
@@ -21,7 +22,15 @@ export let StateSelector: React.StatelessComponent<StateSelectorProps> = ({ clas
     )}
   </DropDownMenu >
 
+/**
+ * As part of State selection, send an event to GA.
+ */
 function onStateChange(value: State, store: Store) {
+  ReactGA.event({
+    category: 'Filter',
+    action: 'Selected a State',
```

Comment:
Selected a State in US-wide dataset. (or smthg like that?)

Reply:
I thought that would be implicit (those are the only datasets where you can perform state selection)